choco/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+    PackageName    = 'yq'
+    Url            = 'https://github.com/mikefarah/yq/releases/download/2.4.1/yq_windows_386.exe'
```

Comment:
Would it be possible to bundle these executables in the package rather than downloading from the web to support those that have air-gapped environments?

Reply:
I believe, the download way is preferred by Choco guidelines.

every Choco package installed on my system uses downloads, so I believe Choco is pretty useless without Internet connection. they do have some kind of 'Choco server for your local net' or something like this that they market for use by business, I believe it's the answer for air-gapped environments for some reason not allowing connections to GitHub.